internal/cli/main.go

```diff
@@ -145,11 +176,25 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		}
 
 	case "goto":
+
+		gotoSet := flag.NewFlagSet("goto", flag.ExitOnError)
```

Comment:
Factor this out into a common function that creates a new `FlagSet` with a help flag: `func newFlagSetWithHelp(name string, errHandling flag.ErrorHandling) (*FlagSet, bool)`